provisioning/service/src/Models/IndividualEnrollment.cs

```diff
@@ -189,7 +189,7 @@ public Attestation Attestation
         /// Capabilities of the device.
         /// </summary>
         [JsonProperty(PropertyName = "capabilities", DefaultValueHandling = DefaultValueHandling.Ignore)]
-        public DeviceCapabilities Capabilities { get; set; }
+        public ProvisioningDeviceCapabilities Capabilities { get; set; }
```

Comment:
Are capabilities only relevant for devices? (The comment seems to suggest that they're only for devices).

Reply:
The only property in it is IsIotEdge, which only applies to devices.

Sure, one day they could extend this to be other things that might also apply to modules, but at this point I doubt that.

Reply:
That said, I'm not against renaming this to `*ClientCapabilities`.